apps/extension/src/core/utils/utils.ts

```diff
@@ -66,3 +66,15 @@ export function formatDate(timestampMs: number): string {
 export function assertNever(value: never): never {
   throw new Error(`Unexpected value: ${value}`)
 }
+
+export function extractExtensionFromURL(url: string): string {
```

Comment:
nit: imo, better to return undefined if we don't have an extension at all than to return an empty string